airflow/providers/amazon/aws/hooks/batch_client.py

```diff
@@ -0,0 +1,503 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+A client for AWS batch services
+
+.. seealso::
+
+    - http://boto3.readthedocs.io/en/latest/guide/configuration.html
+    - http://boto3.readthedocs.io/en/latest/reference/services/batch.html
+    - https://docs.aws.amazon.com/batch/latest/APIReference/Welcome.html
+"""
+
+from random import uniform
+from time import sleep
+from typing import Optional
+
+import botocore.client
+import botocore.exceptions
+from typing_extensions import Protocol
+
+from airflow import AirflowException, LoggingMixin
+from airflow.contrib.hooks.aws_hook import AwsHook
+
+# pylint: disable=invalid-name, unused-argument
+
+
+class BatchProtocol(Protocol):
+    """
+    A typing extension for ``boto3.client('batch') -> botocore.client.Batch``
+    """
+
+    def describe_jobs(self, jobs):
+        """Get job descriptions from AWS batch
```

Comment:
```suggestion
        """
        Get job descriptions from AWS batch
```

Move this to the next line pls.

Reply:
Is this [black](https://github.com/psf/black)s docstring formatting?

I think we should be consistent about it. Can you change it also for the docstrings below?

Reply:
OK; it's not `black` formatting.